### PR TITLE
Add Sympla integrations and change about text

### PR DESCRIPTION
**Pull request by andrelugomes**

### CODE REVIEW

**Review by duduribeiro — CHANGES_REQUESTED** [review]
I suggest to add the url and the div name required by Sympla in the json file. So it could be easily changed in the future events. WDYT?

File: src/views/index.hbs [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
-  <title> DevConf | Somos caipiras e gostamos de tecnologia! </title>
+  <title> {{{info.title}}}</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
   <link rel="stylesheet" href="assets/css/bootstrap.min.css">
   <link rel="stylesheet" href="assets/css/styles.css">
   <link rel="stylesheet" href="assets/css/font-awesome-4.7.0/css/font-awesome.min.css">
   <!-- Open Graph -->
-  <meta property="og:title" content="DevConf - Somos caipiras e gostamos de tecnologia! " />
+  <meta property="og:title" content="DevConf 2018 - Somos caipiras e gostamos de tecnologia! " />
```
Comment (duduribeiro) [inline src/views/index.hbs]: You should use the dynamic year here. Instead of writing 2018 directly

**Review by duduribeiro — CHANGES_REQUESTED** [review]
Another change

**Comment by andrelugomes** [comment]
makes sense!!!!

File: src/views/index.hbs (hunk 1 above)
Comment (andrelugomes) [reply src/views/index.hbs]: I fixed

**Comment by andrelugomes** [comment]
@duduribeiro check this out!

**Review by duduribeiro — APPROVED** [review]
Awesome 💯

**Pull request merged**